### PR TITLE
Add --scalyr-key option to the update subcommand

### PR DESCRIPTION
**Pull request by a1exsh**
By popular request.

### CODE REVIEW

File: planb/update_cluster.py [hunk 1]
```diff
@@ -294,9 +294,15 @@ def build_run_instances_params(
...
     }
+
     docker_image = options.get('docker_image')
     if docker_image:
         user_data_changes['source'] = docker_image
+
+    scalyr_key = options.get('scalyr_key')
+    if scalyr_key:
+        user_data_changes['scalyr_account_key'] = scalyr_key
+
```
Comment (slitsche) [inline planb/update_cluster.py]: Would you like to add a test?

**Comment by a1exsh** [comment]
:+1:

File: planb/update_cluster.py [hunk 2]
```diff
@@ -247,7 +247,7 @@ def terminate_instance(ec2: object, volume: dict, saved_instance: dict):
 
 
 def build_run_instances_params(
-        ec2: object, volume: dict, saved_instance: dict, options: dict) -> dict:
```
Comment (a1exsh) [inline planb/update_cluster.py]: Removed unused parameter ;)

**Comment by slitsche** [comment]
👍

**Pull request merged**